### DISCUSSION TITLE
After applying mol.condense_abbreviations - how do I undo it?

### DISCUSSION THREAD

**Discussion by shyamramesh** [rdkit/rdkit#6862] [Q&A]
rdkitjs - After applying mol.condense_abbreviations - how do I undo it?. I don't want to use smile to get the original mol as I might use other datas of the mol.

**Comment by shyamramesh** [comment]
any update? possible in javascript?

**Comment by ptosco** [comment]
@shyamramesh Please try to avoid cross-posting questions on the same topic in both `rdkit` and `rdkit-js` - ideally, JS-related questions should go to `rdkit-js`.
You cannot undo it. But you can make a copy of the original molecule with `get_mol_copy`, and apply the abbreviations on the copy. So you have both the unabbreviated and the abbreviated version of the same molecule.
